legend-engine-application-query/src/main/java/org/finos/legend/engine/application/query/api/ApplicationQuery.java

```diff
@@ -50,9 +54,16 @@ public class ApplicationQuery
 {
     private final QueryStoreManager queryStoreManager;
 
-    public ApplicationQuery(MongoClient mongoClient)
+    public ApplicationQuery(Object client)
```

Comment:
Lets avoid using Object, and either create two constructors, or make the input parameter to QueryStoreManager